### PR TITLE
fix(site): mount live editor styles only once

### PR DESCRIPTION
**Pull request by coldpour**
### Description

mount live editor styles only once in the site

### Motivation and context

react-live was mounting a <style> tag for every example. this commit turns that
off and loads a style sheet in the head of the site

connects #29

### How has this been tested?

ran the demo

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist

* [x] Renders and functions properly in [all supported browsers](https://github.com/mineral-ui/mineral-ui#browser-support)
* [x] Automated tests written and passing
* [x] [Accessibility](http://webaim.org/intro) and [inclusivity](https://24ways.org/2016/what-the-heck-is-inclusive-design/) considered
* [x] Rendering performance (initial load time & 60fps) and [perceived performance](http://blog.teamtreehouse.com/perceived-performance) considered
* [x] Documentation created or updated

### How does this PR make you feel?

lighter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "eslint": "eslint .",
     "flow": "flow",
     "flow:restart": "npm run flow -- stop; npm run flow",
+    "flow-typed": "flow-typed",
```
Comment (coldpour) [inline package.json]: an npm script to run flow-typed without messing around with node_modules paths

File: packages/site/src/web/index.html [hunk 2]
```diff
@@ -6,15 +6,6 @@
...
     <title>Mineral UI</title>
-    <style>
-      body {
-        margin: 0;
-      }
-      pre.prism-code {
-        white-space: pre-wrap;
-        word-break: break-all;
-      }
-    </style>
```
Comment (coldpour) [inline packages/site/src/web/index.html]: moved into index.css

File: packages/site/src/web/index.js [hunk 3]
```diff
@@ -25,6 +25,8 @@ import App from '../components/App';
 import siteTheme from '../components/siteTheme';
 // $FlowFixMe
 import demos from '{{DEMO_LIST_PATH}}';
+require('./index.css');
+require('react-live/react-live.css');
```
Comment (coldpour) [inline packages/site/src/web/index.js]: these get injected into the <head> in <style> tags in the order they appear here. glamorous also injects a <style> tag and I can't figure out how to get glamorous's not to be first...

File: utils/makeWebpackConfig.js [hunk 4]
```diff
@@ -88,6 +88,10 @@ function getModule() {
         test: /\.js$/,
         exclude: /node_modules/,
         loader: 'babel-loader'
+      },
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader']
```
Comment (coldpour) [inline utils/makeWebpackConfig.js]: style-loader gives us the ability to require css files in our js files. css-loader then interprets the loaded css file. i read that works like a chain of unix shell commands.

File: flow-typed/npm/react-live/react-live.css_vx.x.x.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+ *   'react-live/react-live.css'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'react-live/react-live.css' {
```
Comment (brentertz) [inline flow-typed/npm/react-live/react-live.css_vx.x.x.js]: Why not just add this declaration to the react-live stub?

File: package.json (hunk 1 above)
Comment (brentertz) [inline package.json]: 👍

**Pull request merged**